### ISSUE TITLE
[feature request] Chrome app.

### ISSUE THREAD

**Issue by dhead666** [jez500/chorus#51]
I started getting something done myself but then I though why not leave it to someone who actually know javascript and html and also have it officially published at the Chrome store.

Chromium got pretty great integration with the Gnome desktop as Chromium webapps are shown in the "Gnome Activities" overview and when opened, a webapp seem as regular app with no visible indication that's running in a browser.

I think that a Chrome webapp will be a nice addition to Chorus. 
Basically it would probably take more time to get a proper icon done (round edges please) than to write the webapp.

p.s. 
One of the Chrome webapp features could be support for multiple Chorus clients (just browsing to another address) without any change to Chorus.
